src/backend/utils/indexer.js

```diff
@@ -2,7 +2,7 @@ require('../lib/config');
 
 const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
 
-const { ELASTIC_MAX_RESULTS } = process.env;
+const { MAX_POSTS_PER_PAGE } = process.env;
```

Comment:
This can't be changed without updating our `env` variables in the different sample files we have, e.g., https://github.com/Seneca-CDOT/telescope/blob/master/env.example#L71

Reply:
i only set this as a default if the perPage query wasnt given/undefined. sorry if misunderstood the problem. whats wrong with using the process env constant?

Reply:
The problem is that you've left the `ELASTIC_MAX_RESULTS` orphaned in the env.  If you don't use it anymore, it needs to get deleted.  I think re-using this variable from the env is fine, but you might also want to update the docs in the env files to indicate that this will apply to posts as well as search results.

Reply:
ahhh okay. Maybe we can keep `ELASTIC_MAX_RESULTS` but change its default value in the env for this feature?